libsqlite3-sys/Cargo.toml

```diff
@@ -37,6 +37,12 @@ wasm32-wasi-vfs = []
 [dependencies]
 openssl-sys = { version = "0.9", optional = true }
 
+[target.wasm32-unknown-unknown.dependencies]
+compiler-rt-builtins = "0.1"
```

Comment:
Is it documented which part of the requirements `compiler-rt-builtins` provides vs the stripped down OpenBSD `libc` source?

Reply:
https://github.com/rusqlite/rusqlite/pull/1010#issuecomment-1758124732

Also, I haven't tested things since https://github.com/rust-lang/compiler-builtins/pull/566 got merged, it's possible that might already provide the required functions or get us closer to a cleaner solution in some way.